### PR TITLE
Added files to run integration tests with Stackable CI infrastructure

### PR DESCRIPTION
**Pull request by backstreetkiwi**

### CODE REVIEW

File: .ci/integration-tests/aws-centos-8/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: t2.stackable.tech/v1
+kind: Infra
+template: aws-centos-8
+metadata: 
+  name: spark-operator-integration-tests
```
Comment (maltesander) [inline .ci/integration-tests/aws-centos-8/cluster.yaml:6]: kafka-operator-integration-tests

File: .ci/integration-tests/aws-centos-8/cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      version: 3.5.8
+      servers:
+        roleGroups:
+          default:
+            selector:
+              matchLabels:
+                kubernetes.io/arch: stackable-linux
+            config:
+              adminPort: 12000
+              metricsPort: 9505
```
Comment (maltesander) [inline .ci/integration-tests/aws-centos-8/cluster.yaml:35]: new line for yaml in the end.

**Review by maltesander — CHANGES_REQUESTED** [review]
Just a copy and paste.

File: .ci/integration-tests/aws-centos-8/cluster.yaml (hunk 1 above)
Comment (backstreetkiwi) [reply .ci/integration-tests/aws-centos-8/cluster.yaml]: fixed

File: .ci/integration-tests/aws-centos-8/cluster.yaml (hunk 2 above)
Comment (backstreetkiwi) [reply .ci/integration-tests/aws-centos-8/cluster.yaml]: fixed

**Pull request merged**